### ISSUE TITLE
codebuild failing

### ISSUE THREAD

**Issue by matthewberryman** [AcrossTheCloud/TBA21-helpers#16]
https://github.com/serverless/serverless/issues/6421

**Comment by matthewberryman** [comment]
Failing with following error; I am guessing as it's already installed as a sub-dependency of dev dependencies.

```
Error: EACCES: permission denied, open '/root/.serverlessrc.2104321300' 
    at Object.openSync (fs.js:443:3) 
    at Function.writeFileSync [as sync] (/usr/local/lib/node_modules/serverless/node_modules/write-file-atomic/index.js:212:13) 
    at createConfig (/usr/local/lib/node_modules/serverless/lib/utils/config/index.js:38:19) 
    at getConfig (/usr/local/lib/node_modules/serverless/lib/utils/config/index.js:52:5) 
    at Object.get (/usr/local/lib/node_modules/serverless/lib/utils/config/index.js:99:18) 
    at Object.<anonymous> (/usr/local/lib/node_modules/serverless/lib/utils/isTrackingDisabled.js:5:30) 
    at Module._compile (internal/modules/cjs/loader.js:776:30) 
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10) 
    at Module.load (internal/modules/cjs/loader.js:653:32) 
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12) 
    at Function.Module._load (internal/modules/cjs/loader.js:585:3) 
    at Module.require (internal/modules/cjs/loader.js:690:17) 
    at require (internal/modules/cjs/helpers.js:25:18) 
    at Object.<anonymous> (/usr/local/lib/node_modules/serverless/lib/utils/tracking.js:11:28) 
    at Module._compile (internal/modules/cjs/loader.js:776:30) 
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10) 
npm ERR! code ELIFECYCLE 
npm ERR! errno 1 
npm ERR! serverless@1.48.1 postinstall: `node ./scripts/postinstall.js` 
npm ERR! Exit status 1 
npm ERR!  
npm ERR! Failed at the serverless@1.48.1 postinstall script. 
npm ERR! This is probably not a problem with npm. There is likely additional logging output above. 
 
npm ERR! A complete log of this run can be found in: 
npm ERR!     /root/.npm/_logs/2019-07-22T06_24_36_399Z-debug.log 
 
[Container] 2019/07/22 06:24:36 Command did not exit successfully npm install -g serverless@1.48.1 exit status 1 
[Container] 2019/07/22 06:24:36 Phase complete: POST_BUILD State: FAILED 
[Container] 2019/07/22 06:24:36 Phase context status code: COMMAND_EXECUTION_ERROR Message: Error while executing command: npm install -g serverless@1.48.1. Reason: exit status 1 
```

**Comment by matthewberryman** [comment]
fixed in c83213b
